src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -770,7 +764,7 @@ CHIP_ERROR BLEManagerImpl::StopAdvertising(void)
     {
         // Since DriveBLEState is not called the device is still advertising
         status = rsi_ble_stop_advertising();
-        if (status != RSI_SUCCESS)
```

Comment:
Not really sure why a blanket error status is used to process the stop advertising event?
```
#define SL_STATUS_SI91X_COMMAND_DISALLOWED            (0x4E0C)
if (status == RSI_SUCCESS || status == 0x4E0C)
```